test/IntegrationTests/Helpers/MockZipkinCollector.cs

```diff
@@ -14,6 +14,8 @@
 // limitations under the License.
 // </copyright>
 
+#if NETFRAMEWORK
```

Comment:
We could run the tests in .NET 6.0 and start the test applications in .NET Framework, .NET Core 3.1 etc...

Reply:
The info in which version the test app should be run could be passed via env vars.

Reply:
Sounds perfect, tests should run faster as net6.0 is more performant. Let's see what CI thinks about it.

Reply:
Unfortunately breaks pre-processor flags in `IntegrationTests`.
Makes it hard to run directly in Visual Studio Test Explorer, if target is other than 6.0.

Reply:
Reverting... might need bigger rebuilds / planning. Could cause loads of discomfort in integrations tests development if done wrong.